### PR TITLE
fix bug with DELETE, Add PUT

### PR DESCRIPTION
**Pull request by avolosin**
There was some other weirdness where the project couldn't find the tests.

### CODE REVIEW

File: src/DroolMissle.HttpTester/HttpClientExtensions.cs [hunk 1]
```diff
@@ -39,11 +41,26 @@ public static async Task<HttpRequestCapture> TestPostAsync<T>(this HttpClient cl
             return capture;
         }
 
+        public static async Task<HttpRequestCapture> TestPutAsync<T>(this HttpClient client, string uri, string payload, Encoding encoding, string contentTypeHeader)
+        {
+            var capture = new HttpRequestCapture() { Url = uri, Method = HttpMethod.Post, RequestStartTime = DateTime.UtcNow };
```
Comment (ghost) [inline src/DroolMissle.HttpTester/HttpClientExtensions.cs]: HttpMethod here is `Post` but this is supposed to be a `Put` correct?

File: src/DroolMissle.HttpTester/HttpClientExtensions.cs [hunk 2]
```diff
@@ -39,11 +41,26 @@ public static async Task<HttpRequestCapture> TestPostAsync<T>(this HttpClient cl
...
+            capture.ResponseContent = await response.Content.ReadAsStringAsync();
+            return capture;
+        }
+
         public static async Task<HttpRequestCapture> TestDeleteAsync<T>(this HttpClient client, string uri, string payload, Encoding encoding, string contentTypeHeader)
         {
             var capture = new HttpRequestCapture() { Url = uri, Method = HttpMethod.Delete, RequestStartTime = DateTime.UtcNow };
             var sw = Stopwatch.StartNew();
-            var response = await client.PostAsync(uri, new StringContent(payload, encoding, contentTypeHeader));
+            var response = await client.DeleteAsync(uri);
```
Comment (ghost) [inline src/DroolMissle.HttpTester/HttpClientExtensions.cs:63]: 👍

**Review by ghost — CHANGES_REQUESTED** [review]
please remove .DS_Store from the tests project (and potentially add it to .gitignore)

also, do we know why there are changes to DroolMissle.Tests? (files seemed renamed?)

File: src/DroolMissle.HttpTester/HttpClientExtensions.cs (hunk 1 above)
Comment (avolosin) [reply src/DroolMissle.HttpTester/HttpClientExtensions.cs]: Changed to a PUT

**Comment by avolosin** [comment]
Not sure why the test project was renamed. I just opened it in VS.

**Comment by ghost** [comment]
closed. changes cherry picked and pushed via cleanup branch.

**Pull request closed without merging**